genref/output/md/apiserver-audit.v1.md

```diff
@@ -248,390 +214,323 @@ categories are logged.
 and response bodies from being written to the API audit log.
 This is used as a global default - a value of 'true' will omit the managed fileds,
 otherwise the managed fields will be included in the API audit log.
 Note that this can also be specified per rule in which case the value specified
-in a rule will override the global default.</td>
+in a rule will override the global default.</p>
+</td>
 </tr>
-    
-  
 </tbody>
 </table>
-    
-
 
 ## `PolicyList`     {#audit-k8s-io-v1-PolicyList}
     
 
 
+<p>PolicyList is a list of audit Policies.</p>
 
 
-PolicyList is a list of audit Policies.
-
 <table class="table">
 <thead><tr><th width="30%">Field</th><th>Description</th></tr></thead>
 <tbody>
     
 <tr><td><code>apiVersion</code><br/>string</td><td><code>audit.k8s.io/v1</code></td></tr>
 <tr><td><code>kind</code><br/>string</td><td><code>PolicyList</code></td></tr>
     
-
-  
   
 <tr><td><code>metadata</code><br/>
 <a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#listmeta-v1-meta"><code>meta/v1.ListMeta</code></a>
 </td>
 <td>
-   <span class="text-muted">No description provided.</span>
-   </td>
+   <span class="text-muted">No description provided.</span></td>
 </tr>
-    
-  
 <tr><td><code>items</code> <B>[Required]</B><br/>
 <a href="#audit-k8s-io-v1-Policy"><code>[]Policy</code></a>
 </td>
 <td>
-   <span class="text-muted">No description provided.</span>
-   </td>
+   <span class="text-muted">No description provided.</span></td>
 </tr>
-    
-  
 </tbody>
 </table>
-    
-
 
 ## `GroupResources`     {#audit-k8s-io-v1-GroupResources}
     
 
-
-
 **Appears in:**
 
 - [PolicyRule](#audit-k8s-io-v1-PolicyRule)
 
 
-GroupResources represents resource kinds in an API group.
+<p>GroupResources represents resource kinds in an API group.</p>
+
 
 <table class="table">
 <thead><tr><th width="30%">Field</th><th>Description</th></tr></thead>
 <tbody>
     
-
   
 <tr><td><code>group</code><br/>
 <code>string</code>
 </td>
 <td>
-   Group is the name of the API group that contains the resources.
-The empty string represents the core API group.</td>
+   <p>Group is the name of the API group that contains the resources.
+The empty string represents the core API group.</p>
+</td>
 </tr>
-    
-  
 <tr><td><code>resources</code><br/>
 <code>[]string</code>
 </td>
 <td>
-   Resources is a list of resources this rule applies to.
-
-For example:
+   <p>Resources is a list of resources this rule applies to.</p>
+<p>For example:
 'pods' matches pods.
 'pods/log' matches the log subresource of pods.
-'&lowast;' matches all resources and their subresources.
-'pods/&lowast;' matches all subresources of pods.
-'&lowast;/scale' matches all scale subresources.
-
-If wildcard is present, the validation rule will ensure resources do not
-overlap with each other.
-
-An empty list implies all resources and subresources in this API groups apply.</td>
+'<em>' matches all resources and their subresources.
```

Comment:
nit:  Is the quoted `<em>` element correct?

Reply:
This is incorrect. We still have some nits in parsing the comment string. The upstream [source code](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/apis/audit/v1/types.go#L257-L271) looks like this:

```go
	// Resources is a list of resources this rule applies to.
	//
	// For example:
	// 'pods' matches pods.
	// 'pods/log' matches the log subresource of pods.
	// '*' matches all resources and their subresources.
	// 'pods/*' matches all subresources of pods.
	// '*/scale' matches all scale subresources.
	//
	// If wildcard is present, the validation rule will ensure resources do not
	// overlap with each other.
	//
	// An empty list implies all resources and subresources in this API groups apply.
	// +optional
	Resources []string `json:"resources,omitempty" protobuf:"bytes,2,rep,name=resources"`
```

The problems is that `*` in the comments are sometimes used as a markdown tag, while sometimes used as a literal string. To make things consistent, we will need some protocols for the whole development community.

I'll try if escaping the '*' character before handling the comment to the goldmark will yield better results.

Reply:
👍